pyproject.toml

```diff
@@ -136,6 +136,7 @@ installer = [
     "osmnx>=1.1.0,<1.10",
     "pandas>=1.1.0,<2.3",
     "pyvista[io]>=0.38.0,<0.45",
+    "pyedb>=0.27.0; python_version > '3.7'",
```

Comment:
this may be not needed as pyedb is required already as base dependencies

Reply:
@maxcapodi78 We can remove the python_version specification since we are now python3.8+ compatible (as `pyedb`).
@hui-zhou-a I agree that keeping `pyedb` is a required dependency for installer installation target. However, putting `>=0.27.0` is not enough to ensure that `pyedb` is updated when doing `pip install pyaedt[installer]` unless you are calling for an upgrade explicitly.